openedx/core/lib/new_relic_utils.py

```diff
@@ -0,0 +1,24 @@
+"""
+Contains utilities for working with New Relic.
```

Comment:
One idea I had last night is to make it a static method on ``CourseTabView``. That way it can be called easily from course tab views, but it allows views that haven't been converted to call it too. It also makes it clearer what it is: ``CourseTabView.add_new_relic_course_key_metrics``. What do you think?